CHANGES.md

```diff
@@ -1,5 +1,463 @@
+* made `throw` and `catch` operate on `owned` errors, enabling re-throwing
+  (see https://chapel-lang.org/docs/1.19/primers/errorHandling.html)
+* for records, compiler-generated `=`, `==`, `!=` now require matching types
+  (see 'Record Assignment' and 'Default Comparison Operators' in the spec)
+* `[]`-loops now permit serial execution if parallelism isn't an option
+  (see TODO)
+* default initializers for generic types require formal names to match fields
+  (https://chapel-lang.org/docs/1.19/technotes/initTypeAlias.html)
+
+New Features
+------------
+* added support for directly initializing atomic variables from values
+  (e.g., `var x: atomic int = 10;`)
+* added lifetime annotations for expressing relative lifetimes between arguments
+  (see TODO)
+* certain dereferences of `nil` values are now caught by the compiler
+  (see https://chapel-lang.org/docs/1.19/technotes/nilChecking.html)
+* added support for using type aliases in `new` expressions
+  (see https://chapel-lang.org/docs/1.19/technotes/initTypeAlias.html)
+* added support for the `init=` method, a proposed change to copy initializers
+  (see https://chapel-lang.org/docs/1.19/technotes/initequals.html)
+* added automatic fences for unordered operations at task termination
+
+Feature Improvements
+--------------------
+* added support for compile-time operations on floating point `param` values
+  (see TODO)
+* added config disambiguation for `chpl` arguments via module names
+  (e.g., `-sM.debug=true` can be used to set config `debug` in module `M`)
+* made scan expressions preserve the shape/domain of the input expression
+  (e.g., in `var B = + scan A;`, B will have A's shape and domain)
+* inferred-type arrays defined using range expressions now preserve indices
+  (e.g., `var A = [i in -2..2] i;` now results an array with domain {-2..2})
+* extended `reshape()` to accept loop expressions
+  (e.g., `reshape([i in 1..4] i, {1..2, 1..2})` now works)
+* made default string accesses always return a string
+  (see https://chapel-lang.org/docs/1.19/builtins/String.html#String.string.this)
+* added support for ranges of codepoints and slicing of strings using them
+  (see https://chapel-lang.org/docs/1.19/builtins/String.html)
+* added string.codePoint[s]() to access and iterate over strings by codepoint
+  (see https://chapel-lang.org/docs/1.19/builtins/String.html#String.string.codePoint
+   and https://chapel-lang.org/docs/1.19/builtins/String.html#String.string.codePoints)
+* added string.byte[s]() to access and iterate over strings by byte
+  (see https://chapel-lang.org/docs/1.19/builtins/String.html#String.string.byte
+   and https://chapel-lang.org/docs/1.19/builtins/String.html#String.string.bytes)
+* string casts to integral or floating point types may now contain underscores
+  (e.g., `"1_000_000":int` is now supported)
+* string casts to integral types can now be in base 2, 8, or 16
+  (e.g., `0xabcd`:int` is now supported)
```

Comment:
There should be actual quotation marks around the number, as in the previous line.  And a formatting backtick escaped.

Reply:
Good catch, thanks Paul!